src/screens/SettingsScreen/SettingsScreen.tsx

```diff
@@ -1,26 +1,82 @@
 import React from 'react';
-import { Button } from 'react-native';
+import { Alert, TouchableWithoutFeedback } from 'react-native';
 
-import { useDispatch } from 'react-redux';
+import { useNavigation } from '@react-navigation/native';
+import { useTranslation } from 'react-i18next';
+import { ListItem } from 'react-native-elements';
+import { useDispatch, useSelector } from 'react-redux';
 
 import { NavigationData } from '~/navigation/AppNavigator/SettingsNavigator';
 import { logout } from '~/redux/auth';
+import { RootState } from '~/redux/rootReducer';
+import Colors from '~/theme/Colors';
 
 import { StyledView } from './SettingsScreen.styled';
 
 type Props = NavigationData<'Settings'>;
 
 const SettingsScreen: React.FC<Props> = () => {
   const dispatch = useDispatch();
+  const { username, userImage } = useSelector((state: RootState) => state.auth);
+  const { navigate } = useNavigation();
```

Comment:
One thing to consider: isn't it easier just to take navigation from `props`? It it's already available for a screen, so it might be cleaner to use: `props.navigation.navigate('UserSettings')`.

Reply:
You right, I will change it :+1: